server/conn.go

```diff
@@ -554,20 +558,22 @@ func (cc *clientConn) Run() {
 
 		cc.alloc.Reset()
 		// close connection when idle time is more than wait_timout
-		waitTimeout := cc.getSessionVarsWaitTimeout()
+		waitTimeout := cc.getSessionVarsWaitTimeout(ctx)
 		cc.pkt.setReadTimeout(time.Duration(waitTimeout) * time.Second)
 		start := time.Now()
 		data, err := cc.readPacket()
 		if err != nil {
 			if terror.ErrorNotEqual(err, io.EOF) {
 				if netErr, isNetErr := errors.Cause(err).(net.Error); isNetErr && netErr.Timeout() {
 					idleTime := time.Since(start)
-					log.Infof("con:%d read packet timeout, close this connection, idle: %v, wait_timeout: %v", cc.connectionID, idleTime, waitTimeout)
+					logutil.Logger(ctx).Info("read packet timeout, close this connection",
+						zap.Duration("idle", idleTime),
+						zap.Uint64("waitTimeout", waitTimeout),
+					)
 				} else {
 					errStack := errors.ErrorStack(err)
 					if !strings.Contains(errStack, "use of closed network connection") {
-						log.Errorf("con:%d read packet error, close this connection %s",
-							cc.connectionID, errStack)
+						logutil.Logger(ctx).Error("read packet failed, close this connection", logutil.Error(err), zap.String("stack", errStack))
```

Comment:
I'm not sure will it be easier to use if we wrap all the `zap.String/ zap.Uint/ zap.xxx` to `logutil.String/ logutil.Uint/ loguilt.xx`?